dns-dodo.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=The dns-dodo DNS record updater
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+ExecStart=/usr/bin/dns-dodo update-dns -p
```

Comment:
Added a template unit file and a script to install it as a user unit. This should be sufficient to start the binary installed with `go install` as a systemd service.